setup.py

```diff
@@ -31,7 +31,7 @@
         "banal >= 1.0.6, < 2.0.0",
         "normality >= 2.4.0, < 3.0.0",
         "fakeredis >=2.11.2, < 3.0.0",
-        "sqlalchemy >= 2.0.4, < 3.0.0",
+        "sqlalchemy >= 1.3.2, < 3.0.0",
```

Comment:
Shouldn’t this require at least 1.4 (which is the first version that supports the 2.0 API)?

Reply:
I didn't increase it yet to `1.4` because I still have the lingering issue of tests in `dataset` failing for `SQLAlchemy` version `1.4.49`. Not knowing the reason behind it yet, I didn't want to break these libs for anyone using them in a way that triggers this issue.